rfc6962-bis.xml

```diff
@@ -1853,32 +1853,32 @@ value="Montreal"/>
 
     </references>
 
-    <section title="Supporting V1 and V2 simultaneously" anchor="v1_coexistence">
+    <section title="Supporting v1 and v2 simultaneously" anchor="v1_coexistence">
       <t>
-        Certificate Transparency logs have to be either V1 (conforming to <xref target="RFC6962"/>) or V2 (conforming to this document), as the data structures are incompatible and so a V2 log could not issue a valid V1 SCT.
+        Certificate Transparency logs have to be either v1 (conforming to <xref target="RFC6962"/>) or v2 (conforming to this document), as the data structures are incompatible and so a v2 log could not issue a valid v1 SCT.
       </t>
       <t>
-        CT clients, however, can support V1 and V2 SCTs, for the same certificate, simultaneously, as V1 SCTs are delivered in different TLS, X.509 and OCSP extensions than V2 SCTs.
+        CT clients, however, can support v1 and v2 SCTs, for the same certificate, simultaneously, as v1 SCTs are delivered in different TLS, X.509 and OCSP extensions than v2 SCTs.
       </t>
       <t>
-        V1 and V2 SCTs for X.509 certificates can be validated independently.
-        For precertificates, V2 SCTs should be embedded in the TBSCertificate before submission of the TBSCertificate (inside a V1 precertificate, as described in Section 3.1. of <xref target="RFC6962"/>) so that TLS clients conforming to <xref target="RFC6962"/> but not this document are oblivious to the embedded V2 SCTs.
-        An issuer can follow these steps to produce an X.509 certificate with valid V1 and V2 SCTs:
+        v1 and v2 SCTs for X.509 certificates can be validated independently.
+        For precertificates, v2 SCTs should be embedded in the TBSCertificate before submission of the TBSCertificate (inside a v1 precertificate, as described in Section 3.1. of <xref target="RFC6962"/>) to a v1 log so that TLS clients conforming to <xref target="RFC6962"/> but not this document are oblivious to the embedded v2 SCTs.
+        An issuer can follow these steps to produce an X.509 certificate with valid v1 and v2 SCTs:
```

Comment:
Nit: s/valid/embedded/

Valid is implied, I think.  Not all SCTs are embedded in certs, so this should be explicitly specified.